.github/workflows/labeler.yml

```diff
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
```

Comment:
We are using the snake case convention to GitHub task names.

Reply:
Not a single one had snake case :D but aight, changing all

Reply:
Done!